tests/run_integration_tests.py

```diff
@@ -0,0 +1,144 @@
+import os
+import sys
+from pathlib import Path
+import logging
+import traceback
+import inspect
+import difflib as df
+
+from motleycrew.caсhing import (
+    enable_cache,
+    disable_cache,
+    set_cache_location,
+    set_strong_cache,
+)
+from langchain_community.tools import DuckDuckGoSearchRun
+from motleycrew import MotleyCrew, Task
+from motleycrew.agent.llama_index import ReActLlamaIndexMotleyAgent
+
+CACHE_DIR = "tests/cache"
+DATA_DIR = "tests/data"
+STRONG_CACHE = True
+LOGGING_LEVEL = logging.ERROR
+
+logger = logging.getLogger("integration_test_logger")
+
+
+class IntegrationTestException(Exception):
+    """Integration tests exception"""
+
+    def __init__(self, function_name: str, *args, **kwargs):
+        super(IntegrationTestException, self).__init__(*args, **kwargs)
+        self.function_name = function_name
+
+    def __str__(self):
+        super_str = super(IntegrationTestException, self).__str__()
+        return "{} {}: {}".format(self.__class__, self.function_name, super_str)
+
+
+def single_llama_index_test():
+    """Test example single_llama_index"""
+
+    function_name = inspect.stack()[0][3]
+    search_tool = DuckDuckGoSearchRun()
+    researcher = ReActLlamaIndexMotleyAgent(
+        goal="Uncover cutting-edge developments in AI and data science",
+        tools=[search_tool],
+        verbose=True,
+    )
+    crew = MotleyCrew()
+    # Create tasks for your agents
+    task1 = Task(
+        crew=crew,
+        name="produce comprehensive analysis report on AI advancements",
+        description="""Conduct a comprehensive analysis of the latest advancements in AI in 2024.
+      Identify key trends, breakthrough technologies, and potential industry impacts.
+      Your final answer MUST be a full analysis report""",
+        agent=researcher,
+        documents=["paper1.pdf", "paper2.pdf"],
+    )
+
+    # Instantiate your crew with a sequential process
+    result = crew.run(
+        agents=[researcher],
+        verbose=2,  # You can set it to 1 or 2 to different logging levels
+    )
+
+    content = list(result._done)[0].outputs[0].response
+    excepted_content = read_content(function_name)
+    comparison_results(content, excepted_content)
```

Comment:
Let's import the code from examples (isolating it to a function returning agent output) and call it from here, instead of duplicating